frontend/public/components/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -24,14 +24,14 @@ const overviewQueries = {
   [OverviewQuery.CPU_UTILIZATION]: '((sum(node:node_cpu_utilisation:avg1m) / count(node:node_cpu_utilisation:avg1m)) * 100)[60m:5m]',
   [OverviewQuery.STORAGE_UTILIZATION]: '(sum(node_filesystem_size_bytes) - sum(node_filesystem_free_bytes))[60m:5m]',
   [OverviewQuery.STORAGE_TOTAL]: 'sum(node_filesystem_size_bytes)',
-  [OverviewQuery.PODS_BY_CPU]: 'sort_desc(pod_name:container_cpu_usage:sum)',
+  [OverviewQuery.PODS_BY_CPU]: 'sort_desc(pod_name:container_cpu_usage:sum)*1000^3',
```

Comment:
CPU is measures in CPU seconds, I don’t think multiplying this makes sense. Can you explain this change?

Reply:
humanize functions are trying to find most appropriate unit for the value and it starts from smallest unit, in our case its `ns` https://github.com/openshift/console/blob/master/frontend/public/components/utils/units.js#L43 . We usually end up within `ms` range. I can do the multiplying in code, if you prefer

Reply:
yes that'd be better

Reply:
multiplying distorts the result then though no?

Reply:
updated, multiplying happens in the code now, not in query.

Why would it distort ? Im just moving decimal point

Reply:
I don't know how the units thing works that's in use here, but typically they should get a base unit passed in and calculate the SI-prefix from that

Reply:
Yes, that would be better, but currently `humanizeSeconds` accepts a value in nanoseconds.

It would be good to refactor, but not as part of this PR.